LDAR_Sim/external_sensors/METEC_WIND.py

```diff
@@ -0,0 +1,164 @@
+"""
+An alternative sensor, specifically built to replicate the probability of detection
+    curves provided by a METEC report, which factor in wind speeds
+"""
+# ------------------------------------------------------------------------------
+# Program:     The LDAR Simulator (LDAR-Sim)
+# File:        external_sensors.METEC_WIND
+# Purpose:     METEC specific deployment classes and methods based on variable wind
+#
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the MIT License as published
+# by the Free Software Foundation, version 3.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+#
+# ------------------------------------------------------------------------------
+import numpy as np
+from methods.funcs import measured_rate
+from utils.attribution import update_tag
+from utils.unit_converter import gas_convert
+
+# Constants
+# g/s to kg/hr conversion factor
+GsTOKGh = 3.6
+# m/s to km/hr conversion factor
+MsTOKMh = 3.6
+
+
+def detect_emissions(
+    self,
+    site,
+    covered_leaks,
+    covered_equipment_rates,
+    covered_site_rate,
+    site_rate,
+    venting,
+    equipment_rates,
+):
+    """
+    An alternative sensor module, specifically built to replicate the probability of detection
+    curve of technology whose detection capabilities are reflective of a METEC wind dependent
+    probability of detection curve.
+
+    Utilizes 3 values set as the MDL:
+        mdl = [a, b, c]
+        where
+        a and b : are the PoD curve variables
+        c : represents the floor/minimum cutoff value of the leak rates that the sensor can detect.
+        The cutoff value will be compared to a non-wind normalized rate.
+
+    PoD = 1 / (1 + e ^ (a - b * r ))
```

Comment:
Can we put this formula somewhere in the documentation. People need to know where to get the two numbers and we should make that clear so the external sensors are easy to use.